### PR TITLE
Update Threat_Modeling.md

### PR DESCRIPTION
**Pull request by adamshostack**
Aligned activities list with the Four Question Frame; changed tooling to be more than a single SDK; removed the "you have to find the highest risk things" circular logic

### CODE REVIEW

File: pages/Threat_Modeling.md [hunk 1]
```diff
@@ -142,18 +104,16 @@ argument that can be used to explain and defend the security of an
...
 
-## ThreatModel SDK
+## ThreatModel Tooling
 
-The ThreatModel SDK (<https://github.com/stevespringett/threatmodel-sdk>)
-is a minimalistic Java library that provides a basic
+There is a wide variety of tooling for threat modeling, including OWASP Threat Dragon and "The ThreatModel SDK" (<https://github.com/stevespringett/threatmodel-sdk>)
+which is a minimalistic Java library that provides a basic
 vendor-neutral object model along with the ability to parse reports
 generated from common threat modeling tools. It currently supports:
```
Comment (kingthorin) [inline pages/Threat_Modeling.md:112]: Should probably add https://owasp.org/www-project-threat-dragon/

**Review by kingthorin — CHANGES_REQUESTED** [review]
Please add yourself to as an author or contributor in the front matter at L5/6

File: pages/Threat_Modeling.md (hunk 1 above)
Comment (adamshostack) [reply pages/Threat_Modeling.md]: Added link; I don't think I've added enough to list myself as a contributor so will pass on that.

**Comment by kingthorin** [comment]
Please add yourself to as an author or contributor in the front matter at L5/6

**Comment by adamshostack** [comment]
I don't think this update is big enough for me to add myself as a contributor.

**Comment by kingthorin** [comment]
Ok, it all counts, but I'm not going to argue.

**Review by kingthorin — APPROVED** [review]

**Comment by adamshostack** [comment]
Thanks! :)

**Comment by kingthorin** [comment]
Np, thank you for the touch up :tada:

**Pull request merged**